packages/eslint-config-airbnb-base/rules/es6.js

```diff
@@ -10,9 +10,6 @@ module.exports = {
       'objectLiteralDuplicateProperties': false
```

Comment:
I can't find these options [here](http://eslint.org/docs/user-guide/configuring#specifying-parser-options). Is it useful?